main.go

```diff
@@ -262,7 +270,13 @@ func run(args []string, stdin io.Reader, stderr io.Writer, stdout io.Writer) err
 
 		f, comments, ok := patchRunner.Apply(filename, f)
 		// If at least one patch didn't match, there's nothing to do.
+		// If --print was passed, print the contents out as-is.
```

Comment:
Please fix the comment as there is no flag `--print`, but `--print-only`

Reply:
Oops. Yes.